src/redux/categories/categoriesSlice.js

```diff
@@ -4,7 +4,7 @@ const initialState = {
   categories: [],
```

Comment:
- [ ] Based on the project requirement, you are suppose to add the initial book object to the BookSlice as shown below 👇🏾 


```javascript
```javascript
import { createSlice } from "@reduxjs/toolkit"

const initialState = {
  books: [
    {
      item_id: "item1",
      title: "The Great Gatsby",
      author: "John Smith",
      category: "Fiction",
    },
    {
      item_id: "item2",
      title: "Anna Karenina",
      author: "Leo Tolstoy",
      category: "Fiction",
    },
    {
      item_id: "item3",
      title: "The Selfish Gene",
      author: "Richard Dawkins",
      category: "Nonfiction",
    },
  ],
}

export const booksSlice = createSlice({
  name: "books",
  initialState,
  reducers: {
    addBook: (state, action) => {
      state.books = [...state.books, action.payload]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})

export const { addBook, removeBook } = booksSlice.actions

export default booksSlice.reducer
```
```

Reply:
This change has been done well